app/containers/TodoApp/index.js

```diff
@@ -0,0 +1,92 @@
+/* @flow */
+
+import React from 'react';
+import List from '../../components/List';
+import TodoForm from './TodoForm';
+
+type Props = {
+  data: any,
+};
+
+type State = {
+  todos: any,
+  nextId: number,
```

Comment:
is nextId the id of the todo to be added? 
can we have short comments/descriptions on state vars?